buildSrc/src/main/kotlin/scripts/infrastructure.gradle.kts

```diff
@@ -1,7 +1,13 @@
 package scripts
 
+import scripts.Variants_gradle.*
 import java.util.*
 
+private object Default {
+    val BUILD_TYPE = BuildTypes.DEBUG.capitalize()
+    val BUILD_FLAVOR = ProductFlavors.DEV.capitalize()
```

Comment:
can they be `const`?

Reply:
They cannot. Gradle complains because this comes from autogenerated compiled scripts